### PR TITLE
Game

### PR DESCRIPTION
**Pull request by malikasen**
Please review my game project

### CODE REVIEW

File: game-project/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (Shaylan7) [inline game-project/index.html:1]: Very nicely formatted HTML with proper indentation

File: game-project/script.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+function getRandomNumber(min, max) {
```
Comment (Shaylan7) [inline game-project/script.js:1]: Please add comments

File: game-project/script.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+var label3 = document.querySelector('#guesses');
+
+var checkBox = document.getElementById('generate');
+checkBox.addEventListener('click', function() {
+  var isPressed = checkBox.getAttribute('aria-pressed');
+  if (isPressed === 'false') {
+    checkBox.setAttribute('aria-pressed','true');
+    // checkBox.className = 'checked';
+  } else {
+    checkBox.setAttribute('aria-pressed','false');
```
Comment (Shaylan7) [inline game-project/script.js:65]: great use of accessibility attributes

**Review by Shaylan7 — APPROVED** [review]
Looks good! Approved

File: game-project/script.js (hunk 2 above)
Comment (malikasen) [reply game-project/script.js:1]: will do

**Pull request merged**